pkgs/development/python-modules/monkeyhex/default.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage
+, fetchPypi
+, future
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "monkeyhex";
+  version = "1.7.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5ba913df664c34f3ce53916c83872fddf750adc78a0b0ecdd316ac3e728bb019";
+  };
+
+  propagatedBuildInputs = [ future ];
+
```

I would be honest about the amount of testing the package has, and then add pythonImportsCheck to make sure that it doesn't immediate error when someone tries to use it.


```suggestion

  # no tests in repo
  doCheck = false;
  
  # verify imports still work
  pythonImportsCheck = [ "monkeyhex" ];
```